packages/flutter_tools/test/integration.shard/overall_experience_test.dart

```diff
@@ -0,0 +1,556 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// The purpose of this test is to verify the end-to-end behavior of
+// "flutter run" and other such commands, as closely as possible to
+// the default behavior. To that end, it avoids the use of any test
+// features that are not critical (-dflutter-test being the primary
+// example of a test feature that it does use). For example, no use
+// is made of "--machine" in these tests.
+
+// There are a number of risks when it comes to writing a test such
+// as this one. Typically these tests are hard to debug if they are
+// in a failing condition, because they just hang as they await the
+// next expected line that never comes. To avoid this, here we have
+// the policy of looking for multiple lines, printing what expected
+// lines were not seen when a short timeout expires (but timing out
+// does not cause the test to fail, to reduce flakes), and wherever
+// possible recording all output and comparing the actual output to
+// the expected output only once the test is completed.
+
+// To aid in debugging, consider passing the `debug: true` argument
+// to the runFlutter function.
+
+// @dart = 2.8
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:meta/meta.dart';
+import 'package:process/process.dart';
+
+import '../src/common.dart'; // ignore: import_of_legacy_library_into_null_safe
```

Comment:
I don't think you need this ignores if this file is opted out with the 2.8 above

Reply:
oh right, i added these before i realised i needed to just opt this file out